cub/cub/device/dispatch/tuning/tuning_reduce_by_key.cuh

```diff
@@ -717,6 +717,12 @@ struct sm100_tuning<KeyT, AccumT, primitive_op::yes, primitive_key::yes, primiti
   static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
 };
 
+// I16, F32, I32 regresses, default it back.
+template <class KeyT>
+struct sm100_tuning<KeyT, float, primitive_op::yes, primitive_key::yes, primitive_accum::yes, key_size::_2, accum_size::_4>
+    : sm90_tuning<KeyT, float, primitive_op::yes, primitive_key::yes, primitive_accum::yes, key_size::_4, accum_size::_1>
```

Comment:
Q: are you sure we should default back to `accum_size::_1`?

Reply:
fixed